docker-compose/tomcat-manager/docker-compose.yml

```diff
@@ -0,0 +1,241 @@
+version: '3.3'
+services:
+  postgres:
+    image: 'postgres:9'
+    container_name: postgres
+#    ports:
+#      - '5432:5432'
+    environment:
+      - 'POSTGRES_MULTIPLE_DATABASES=apiman,${POSTGRES_KEYCLOAK_DATABASE}'
+      - 'POSTGRES_USER=${POSTGRES_USER}'
+      - 'POSTGRES_PASSWORD=${POSTGRES_PASSWORD}'
+    volumes:
+      - type: bind
+        source: $PWD/../common/create-multiple-postgresql-databases.sh
+        target: /docker-entrypoint-initdb.d/create-multiple-postgresql-databases.sh
+      - type: volume
+        source: postgres_data
+        target: /var/lib/postgres
+      - type: volume
+        source: postgres_logs
+        target: /var/log/postgres
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "10m"
+        max-file: "10"
+        mode: non-blocking
+#    labels:
+#      - traefik.http.routers.postgres-router.tls=true
+#      - traefik.http.routers.postgres-router.entrypoints=https
+#      - traefik.http.routers.postgres-router.rule=PathPrefix(`/postgres`)
+#      - traefik.http.routers.postgres-router.service=postgres-service
+#      - traefik.http.services.postgres-service.loadbalancer.server.port=5432
+#      - traefik.http.routers.postgres-router.middlewares=test-stripprefix
+#      - traefik.http.middlewares.test-stripprefix.stripprefix.prefixes=/postgres
+    restart: always
+
+  elasticsearch:
+    image: 'elasticsearch:7.13.3'
+    container_name: elasticsearch
+    environment:
+      - discovery.type=single-node
+      - bootstrap.memory_lock=true
+      - ES_JAVA_OPTS=-Xms${ELASTICSEARCH_JAVA_MEMORY} -Xmx${ELASTICSEARCH_JAVA_MEMORY}
+      - xpack.security.enabled=false
+#    ports:
+#      - '${ELASTICSEARCH_PORT}:${ELASTICSEARCH_PORT}'
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+      nofile:
+        soft: 65536
+        hard: 65536
+      nproc:
+        soft: 4096
+        hard: 4096
+    volumes:
+      - type: volume
+        source: elasticsearch_data
+        target: /usr/share/elasticsearch/data
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "10m"
+        max-file: "10"
+        mode: non-blocking
+#    labels:
+#      - traefik.http.routers.elasticsearch-router.tls=true
+#      - traefik.http.routers.elasticsearch-router.entrypoints=https
+#      - traefik.http.routers.elasticsearch-router.rule=PathPrefix(`/elasticsearch`)
+#      - traefik.http.routers.elasticsearch-router.service=elasticsearch-service
+#      - traefik.http.services.elasticsearch-service.loadbalancer.server.port=9200
+    restart: always
+
+  keycloak:
+    image: 'jboss/keycloak:15.0.0'
```

Comment:
Let's consider externalising the version and bumping to latest release. 

I think they prefer to release on quay.io now rather than Docker Hub.